providers/__tests__/notificationProvider.test.mjs

```diff
@@ -0,0 +1,39 @@
+import { render, act, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import {
+  NotificationProvider,
+  NotificationDispatch,
+} from '@/providers/NotificationProvider';
+
+jest.useFakeTimers();
```

Comment:
Shouldn't this be inside the describe? I recall Jest by default runs each test file isolated, but... maybe better putting this inside the describe